File: Traductor_VB_JAVA/bin/Debug/Archivo1.java
import java.util.Scanner;
public class Archivo1{
static Scanner sc = new Scanner(System.in);
public static void main(String[] Args){
SentenciaSwtich(1);
 SentenciaIf(63);
}
public static void SentenciaSwtich (int nota){
switch(nota){
case 1:
System.out.println("Su numero es 1") ;
break;
case 2:
System.out.println("Su numero ingresada es 2") ;
break;
case 3:
System.out.println("Su numero ingresada es 3") ;
break;
default:System.out.println("ES OTRA NUMERO") ;
}
}
public static void SentenciaIf (int nota){
System.out.println("Sunota es "+nota) ;
 if(nota<=100&&nota>=0){
if(nota<=100&&nota>=80){
System.out.println("Gano con buena nota") ;
System.out.println("Gano con buena nota") ;
}
}else {
System.out.println("La nota no es valida") ;
}

}

}

File: Entradas Practica1/Archivo1.vb
Module Archivo1

    Sub Main()
        SentenciaSwtich(1)
        SentenciaIf(63)
    End Sub

    Sub SentenciaSwtich(ByVal nota As Integer)

        Select Case nota
            Case 1
                Console.WriteLine("Su numero ingresada es 1")
            Case 2
                Console.WriteLine("Su numero ingresada es 2")
            Case 3
                Console.WriteLine("Su numero ingresada es 3")
            Case Else
                Console.WriteLine("ES OTRA NUMERO")
        End Select
    End Sub

    Sub SentenciaIf(ByVal nota As Integer)
        Console.WriteLine("Sunota es " & nota)
        If nota <= 100 And nota >= 0 Then
            If nota <= 100 And nota >= 80 Then
                Console.WriteLine("Gano con buena nota")
            ElseIf nota < 80 And nota > 64 Then
                Console.WriteLine("Gano")
            ElseIf nota < 65 And nota > 60 Then
                Console.WriteLine("Ufff, casi pierde, pero gano :D")
            End If
        Else
            Console.WriteLine("La nota no es valida")
        End If
    End Sub


End Module
